stock_picking_batch_oca/views/stock_batch_picking.xml

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!-- stock.picking.batch form view -->
+    <record model="ir.ui.view" id="stock_batch_picking_form">
+        <field name="name">stock.picking.batch.form</field>
+        <field name="model">stock.picking.batch</field>
```

Comment:
```suggestion
        <field name="model">stock.picking.batch</field>
        <field name="priority" eval="10"/>
```
With default priority the below method returns stock_picking_batch.stock_picking_batch_form:
https://github.com/i-vyshnevska/stock-logistics-workflow/blob/fdd8ba5cbdaa33cc587317d027830105128b7eae/stock_picking_batch_oca/wizard/batch_picking_creator.py#L158